server/src/server/kb/concept/Serialiser.java

```diff
@@ -78,26 +72,23 @@ public VALUE serialise(VALUE value) {
 
         @Override
         public VALUE deserialise(VALUE value) {
-            return value;
+            return  value;
         }
     }
 
-    public static class Date extends Serialiser<LocalDateTime, Long> {
-
-        Date() {
-            super();
-        }
+    public static class Date extends Serialiser<java.time.LocalDateTime, Long> {
 
         @Override
         public java.lang.Long serialise(LocalDateTime value) {
```

Comment:
this seems to be remnant from old code, but might as well remove `java.lang.` prefix too.